SU2_DEF/src/SU2_DEF.cpp

```diff
@@ -230,11 +230,34 @@ int main(int argc, char *argv[]) {
 
   for (iZone = 0; iZone < nZone; iZone++){
 
+    /*--- Initialize total deformation of design variables to zero ---*/
+    su2double **TotalDeformation;
```

Comment:
`Use su2activematrix instead of su2double**.`  from @pcarruscag prob still applies here in the new home of TotalDeformation?
You then would most likely
```c++
su2activematrix TotalDeformation;
TotalDeformation.resize(nDV,nDV_Value) = su2double(0.0);
```
... that resizing in the loop is not that nice, so I would like to hear pedros opinion before anything is changed here